packages/@o3r/core/schematics/rule-factories/generic-updates/index.ts

```diff
@@ -0,0 +1,27 @@
+import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+
+/**
+ * Generic basic updates to be done on a repository
+ */
+export function genericUpdates(): Rule {
+
+  const updateGitIgnore = (tree: Tree, _context: SchematicContext) => {
+    // update gitignore
+    if (tree.exists('/.gitignore')) {
+      let gitignore = tree.readText('/.gitignore');
+
+      gitignore = gitignore.replace(/(\r\n|\n)\/node_modules(\r\n|\n)/, '$1node_modules$2');
```

Comment:
when working on the it-tests, I also had issues with the `/dist` folder only being ignored at root level

Reply:
I'll take that into consideration too. I'll do an update.